java/dataset/src/test/java/org/apache/arrow/dataset/TestFragmentScanOptions.java

```diff
@@ -165,4 +165,44 @@ public void testCsvConvertOptionsNoOption() throws Exception {
       assertEquals(3, rowCount);
     }
   }
+
```

Comment:
Can we add parameterized tests for the `setCsv*Options` functions?

Reply:
Plus we might want to check whether all the options are properly used.

Reply:
Then we will need a lot of tests to check whether all the options are properly used. Do you think we should do that? Now I only add test to test one of CsvxxxOptions are properly used. @lidavidm @vibhatha

Reply:
What I meant was just to check if the passed options are retrieved as expected, just a config test. Not to check the functionality.

Reply:
Ideally we'd have a brief test of each option, I think